soccer/src/soccer/strategy/agent/position/runner.cpp

```diff
@@ -0,0 +1,266 @@
+        // DEFINE SEARCHING BEHAVIOR
+        SPDLOG_INFO("runner: moving");
+        rj_geometry::Point target_pt = rj_geometry::Point{2, 5};
+        rj_geometry::Point target_vel{0.0, 0.0};
+        planning::LinearMotionInstant target{target_pt, target_vel};
+        rj_geometry::Point robot_position =
+            world_state()->get_robot(true, robot_id_).pose.position();
+        double dist_to_target = robot_position.dist_to(target_pt);
+        SPDLOG_INFO("dist: %lf", dist_to_target);
+        if (dist_to_target < 0.1) {
+            SPDLOG_INFO("at target for USIDE");
+            at_corner = true;
+        } 
+        SPDLOG_INFO("at corner? %s", at_corner?"true":"false");
+        //SPDLOG_INFO("robot position: " + robot_position);
+        auto move_cmd = planning::MotionCommand{"path_target", target};
+        intent.motion_command = move_cmd;
+        return intent;
+    } else if (current_state_ == MOVING_RSIDE) {
+        // DEFINE SEARCHING BEHAVIOR
+        SPDLOG_INFO("runner: moving");
+        rj_geometry::Point target_pt = rj_geometry::Point{2, 4};
+        rj_geometry::Point target_vel{0.0, 0.0};
+        planning::LinearMotionInstant target{target_pt, target_vel};
+        rj_geometry::Point robot_position =
+            world_state()->get_robot(true, robot_id_).pose.position();
+        double dist_to_target = robot_position.dist_to(target_pt);
+        SPDLOG_INFO("dist: %lf", dist_to_target);
+        if (dist_to_target < 0.1) {
+            SPDLOG_INFO("at target for RSIDE");
+            at_corner = true;
+        } 
+        SPDLOG_INFO("at corner? %s", at_corner?"true":"false");
+        //SPDLOG_INFO("robot position: " + robot_position);
+        auto move_cmd = planning::MotionCommand{"path_target", target};
+        intent.motion_command = move_cmd;
+        return intent;
+    } else if (current_state_ == MOVING_DSIDE) {
+        // DEFINE SEARCHING BEHAVIOR
+        SPDLOG_INFO("runner: moving");
+        rj_geometry::Point target_pt = rj_geometry::Point{1, 4};
+        rj_geometry::Point target_vel{0.0, 0.0};
+        planning::LinearMotionInstant target{target_pt, target_vel};
+        rj_geometry::Point robot_position =
+            world_state()->get_robot(true, robot_id_).pose.position();
+        double dist_to_target = robot_position.dist_to(target_pt);
+        SPDLOG_INFO("dist: %lf", dist_to_target);
+        if (dist_to_target < 0.1) {
+            SPDLOG_INFO("at target for DSIDE");
+            at_corner = true;
+        } 
+        SPDLOG_INFO("at corner? %s", at_corner?"true":"false");
+        //SPDLOG_INFO("robot position: " + robot_position);
+        auto move_cmd = planning::MotionCommand{"path_target", target};
+        intent.motion_command = move_cmd;
+        return intent;
+    } 
+
+    // should be impossible to reach, but this is an EmptyMotionCommand
+    return std::nullopt;
+}
+
+void Runner::receive_communication_response(communication::AgentPosResponseWrapper response) {
```

Comment:
again, none of this scoring code should have been included. I assume you copy-pasted from `offense` or some other position. make sure to understand any code you copy-paste and see if it is necessary.